### PR TITLE
Improvement/s3 utils 118 improve verification replication util

### PR DESCRIPTION
**Pull request by naren-rajendran**

### CODE REVIEW

**Review by tmacro — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1290,26 +1291,111 @@ The output of the script consists of JSON log lines.
...
+  "bucket":"src-bucket",
+  "key":"pref1/key404",
+  "level":"error",
+  "message":"error getting metadata",
+  "hostname":"scality.local",
+  "pid":11398
+}
 ```
 
 The script also logs a progress update (a summary), every 10 seconds by default,
```
Comment (jonathan-gramain) [inline README.md:1348]: The following JSON example does not look related to progress update, but to final completion of the script.

**Review by jonathan-gramain — APPROVED** [review]

**Review by alexanderchan-scality — APPROVED** [review]
